halo/attest/keeper/keeper.go

```diff
@@ -796,7 +797,7 @@ func (k *Keeper) deleteBefore(ctx context.Context, height uint64) error {
 		// once we start catching up.
```

Comment:
maybe add a comment here, something like: `also don't delete anything before we have at least one approved attestation`

Reply:
Other option is to remove the boolean and just return `0` if not found, then we will not delete anything either

Reply:
Comment added, thanks!